### PR TITLE
HAAR-848 email domain api corrections

### PR DESCRIPTION
**Pull request by AlTurner-MOJ**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt [hunk 1]
```diff
@@ -1,70 +1,55 @@
 package uk.gov.justice.digital.hmpps.externalusersapi.resource
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt:1]: Your endpoints are missing swagger docs

File: src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainControllerTest.kt [hunk 2]
```diff
@@ -78,47 +72,4 @@ class EmailDomainControllerTest {
     assertEquals("Fred", eventDetails.firstValue.getValue("username"))
     assertEquals(id.toString(), eventDetails.firstValue.getValue("id"))
   }
-
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainControllerTest.kt:81]: Do we need additional tests - for some of the ones now removed...
e.g. delete domain
validation failure

File: src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt (hunk 1 above)
Comment (AlTurner-MOJ) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt:1]: If you're ok i'd prefer to break this out into a separate ticket which i'll pick up last as possible they may end up changing again?

Comment (karenmillermoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt:1]: sure

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**